tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -1372,26 +1362,23 @@ TEST (tensorSinkTest, signalRate)
   /** check received buffers */
   EXPECT_TRUE (g_test_data.received < num_buffers);
 
-  /** check caps name */
-  EXPECT_STREQ (g_test_data.caps_name, "other/tensor");
-
   /** check timestamp */
   EXPECT_FALSE (g_test_data.invalid_timestamp);
 
   /** check caps and config for tensor */
   {
     GstCaps *caps;
     GstStructure *structure;
-    GstTensorConfig config;
+    GstTensorsConfig config;
 
-    caps = gst_tensor_caps_from_config (&g_test_data.tensor_config);
+    caps = gst_tensors_caps_from_config (&g_test_data.tensors_config);
     structure = gst_caps_get_structure (caps, 0);
 
-    EXPECT_TRUE (gst_tensor_config_from_structure (&config, structure));
-    EXPECT_TRUE (gst_tensor_config_is_equal (&config, &g_test_data.tensor_config));
-    EXPECT_TRUE (gst_caps_is_equal (g_test_data.current_caps, caps));
+    EXPECT_TRUE (gst_tensors_config_from_structure (&config, structure));
+    EXPECT_TRUE (gst_tensors_config_is_equal (&config, &g_test_data.tensors_config));
```

Comment:
I don't understand intention of these lines.
Get a cap from a config, get config from the cap and compare it to the original config again?
And why do you remove 1392 line?

Reply:
Considered to compare configuration only, not caps.
As you mentioned, this block is just for conversion between caps and config (config to caps and caps to config).
IMO, this is old testcase to check config struct, and unnecessary now :)

Reply:
Thanks for the comment.